tests/pytests/unit/states/mysql/test_user.py

```diff
@@ -60,7 +61,10 @@ def test_present():
         with patch.object(salt.utils.data, "is_true", mock_t):
             comt = "User frank@localhost is already present with passwordless login"
             ret.update({"comment": comt, "result": True})
-            assert mysql_user.present(name, allow_passwordless=True) == ret
+            assert (
```

Comment:
**issue (blocking):** I don't see how the test changes validate anything other than the fact that the state accepts the parameter. Can you add a check for the presence of the proper SQL command in the underlying call args?

Reply:
I have added a version check to check whether it is using minimum version that supports comment or not

Reply:
https://dev.mysql.com/doc/refman/8.0/en/create-user.html#create-user-comments-attributes

Reply:
I see that you added some version checking in the module code, but I was talking about the pytest tests. You're passing the new `comments` parameter, but there aren't any checks to see if it does what you expect. Perhaps @waynew can take a look during a Test Workshop and give you some ideas on how to test that the underlying call args were modified in a way you'd expect.

Reply:
Oh now I get it, sorry I thought version checking was enough, I will add appropriate pytest tests, thank you for your comment